api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1164,6 +1164,10 @@ message UpstreamConnectionOptions {
 
   // If set then set SO_KEEPALIVE on the socket to enable TCP Keepalives.
   core.v3.TcpKeepalive tcp_keepalive = 1;
+
+  // If true, Envoy will make the interface name of the local address for the upstream connection
+  // available for debugging purposes. Defaults to false due to performance concerns.
+  bool set_local_interface_name_on_upstream_connections = 2;
```

Comment:
Should this use `google.protobuf.BoolValue` instead, so that each xDS client can decide its own default value?

Reply:
I think that as it stands the default should always be false. But I am happy to switch to `google.protobuf.BoolValue` if that is the convention we should go with.